### DISCUSSION TITLE
希望能增加个功能可以自己选择当前页面要进行翻译的区域，并可以保存这个设置，下次翻译时可以应用之前设置的翻译区域

### DISCUSSION THREAD

**Discussion by not502** [immersive-translate/immersive-translate#2319] [Ideas]
### 新功能 | New Feature

可以自己选择当前页面要进行翻译的区域，并可以保存这个设置，下次翻译时可以应用之前设置的翻译区域

### 为什么您想要添加这些功能 | Why you want to add these features

在使用翻译时经常会出现把所有内容都进行翻译，但是我只想对某一块进行翻译，比如下面这张图，我只想要在框选范围内翻译
![image](https://github.com/user-attachments/assets/f6320d57-ed0d-4097-9886-7f2bcd54d3cd)

### 提议 | Proposal

可以做成类似`语雀插件`的选取剪藏那种选取效果，然后对选取的区域进行翻译，并可以保存选取设置下次就不用再重新选取了
![image](https://github.com/user-attachments/assets/2d734aa7-0c58-4b87-8e66-251e7046f63a)

**Comment by function2-llx** [comment]
目前可以在高级自定义选项中可以配置翻译区域，[参考文档](https://immersivetranslate.com/zh-Hans/docs/advanced/#rules)。
